recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    version = "3.6.1"
```

Comment:
```suggestion
```
generic recipe files, which are in "all" folder, should not contain a version declared.

Reply:
removed